docs/usage.md

```diff
@@ -424,6 +424,10 @@ Therefore currently nf-core/taxprofiler does not run ganon on data specified as
 
 KMCP is only suitable for short-read metagenomic profiling, with much lower sensitivity on long-read datasets. Therefore, nf-core/taxprofiler does not currently run KMCP on data specified as being sequenced with `OXFORD_NANOPORE` in the input samplesheet.
 
+##### sylph
```

Comment:
Missing all other sections (e.g. database sheet section)

- [ ] Entry in example database sheet (full database sheet section)
- [ ] Entry in 'Expected' paths bullet points (full database sheet section)
- [x] Entry in 'Classification and profiling' tips section
- [ ] Relevant entries in Post Processing section (Visualisation, multi-table generation)
- [ ] Entry in 'Building custom databases' docs/usage/tutorial.md